Remove the "-fvisibility=hidden" compiler option, otherwise we have problems
trying to import the Crypto.Hash.SHA256 module, causing the remote file
components to fail - bug 89844.

Index: /Users/toddw/as/komodo-devel/contrib/pycrypto/setup.py
===================================================================
--- pycrypto/setup.py	(revision 62953)
+++ pycrypto/setup.py	(working copy)
@@ -148,6 +148,11 @@
                 self.__add_compiler_option("-ftest-coverage")
                 self.compiler.libraries += ['gcov']
 
+        # ACTIVESTATE: Patch for building on Mac OS X - bug 89844.
+        if sys.platform == 'darwin':
+            self.__remove_compiler_option("-fvisibility=hidden")
+        # ACTIVESTATE: end of patch.
+
         # Call the superclass's build_extensions method
         build_ext.build_extensions(self)
 
